spec/features/users/register_spec.rb

```diff
@@ -22,6 +22,10 @@
     end
 
     it 'can register a new user if all required attributes are provided', :vcr do
+      allow(BackEndFacade).to receive(:get_user_friends).with(@user.id).and_return([])
+      allow(BackEndFacade).to receive(:get_user_gyms).with(@user.id).and_return([])
+      allow(BackEndFacade).to receive(:get_user_events).with(@user.id).and_return([])
```

Comment:
These stubs needed to be added to get the register spec to pass.

Reply:
Added on my branch, same problem